controllers/managedocs_controller.go

```diff
@@ -1149,6 +1163,29 @@ func (r *ManagedOCSReconciler) reconcileIngressNetworkPolicy() error {
 	return nil
 }
 
+func (r *ManagedOCSReconciler) reconcileAdditionalMetrics() error {
+	if r.checkUninstallCondition() {
+		metrics.ODFAddonPhase.With(
+			prometheus.Labels{
+				"status": "Uninstalling",
+			},
+		).Set(1)
+	} else if r.areComponentsReadyForUninstall() {
```

Comment:
invoking this method is misleading though it satisfies the need, can we modify the methodname to `areComponentsReady`?

Reply:
Yes, I'll change the method name